### PR TITLE
feat: move audio player to measure when clicking song overview

### LINKED ISSUE #32: Selecting same measure multiple times doesn't work
To reproduce: 
1. Click on a measure in the section overview: the song overview jumps to that measure
2. Let the song overview move to a different part by playing the song for a bit
3. Click the same measure again in the section overview: the song overview does not jump back to that measure.

### PR DESCRIPTION
**Pull request by jasconias**
Just a small PR to test the waters :)
Note: this does not yet fix https://github.com/tscz/transcribe/issues/32

### CODE REVIEW

File: src/components/audioManagement/audioPlayer.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ interface Player {
   getCurrentTime: () => number;
   getDuration: () => number;
   seek: (time: number) => void;
+  setSeconds: (seconds: number) => void;
```
Comment (tscz) [inline src/components/audioManagement/audioPlayer.ts:13]: The seek() method is already doing the job of setSeconds. Furthermore it throws Peaks.js events such that the waveform is informed about visually changing the playhead.

File: src/components/audioManagement/audioManagement.tsx [hunk 2]
```diff
@@ -130,6 +130,7 @@ class AudioManagement extends React.Component<AllProps> {
       prevProps.loopStart !== this.props.loopStart ||
       prevProps.loopEnd !== this.props.loopEnd
     ) {
+      this.getPlayer()?.setSeconds(this.props.loopStart);
```
Comment (tscz) [inline src/components/audioManagement/audioManagement.tsx:133]: It would be better to simply use `this.getPlayer().seek(this.props.loopStart);` here.

**Review by tscz — CHANGES_REQUESTED** [review]

**Comment by tscz** [comment]
Also sorry, in the last two days I resolved some technical depths and made changes (mostly formatting, but still merge conflicting) in lots of files. Maybe it is easier to create this branch new from current master tip and add the change.

File: src/components/audioManagement/audioPlayer.ts (hunk 1 above)
Comment (jasconias) [reply src/components/audioManagement/audioPlayer.ts:13]: Ah, didn't notice that!

**Pull request closed without merging**